### PR TITLE
[OFFAPPS-1211] Invoke a ZAF resize on "app.activated"

### PR DESCRIPTION
**Pull request by adrianbonnici**
## Description
At a high level this PR aims to invoke a ZAF resize when the app is `activated` to ensure that the content is always shown in respect to the iframe's `scrollHeight`.

## Notes
I locked the `handlebars` version to `4.0.5` in the `package.json` to match the version which is being loaded from the `CDN` in `src/templates/iframe.html`. 

## References
[JIRA ticket](https://zendesk.atlassian.net/browse/OFFAPPS-1211)

## Risks
- [Low] The Five Most Recent app is not resizing correctly if an agent navigates away from the active ticket while the app is still loading results in the UI.

### CODE REVIEW

**Comment by dadah89** [comment]
https://github.com/zendesk/five_most_recent_app/pull/47

**Comment by adrianbonnici** [comment]
> #47

@dadah89 shall I wait for you to merge your PR?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Anyone specific?
 Unless there's a good reason not to:
 @zendesk/apps-migration
 If translation-related:
-@zendesk/i18n
+@zendesk/localization
```
Comment (adrianbonnici) [inline .github/pull_request_template.md:32]: @dadah89 I can see your change here after you cherry-picked the commit from https://github.com/zendesk/five_most_recent_app/pull/47/. I can go ahead with including this change in this PR 👍

File: src/javascripts/app.js [hunk 2]
```diff
@@ -124,11 +136,18 @@ class App {
...
-      width: '100%',
-      height: body.scrollHeight
-    })
+    const height = (body && body.scrollHeight) || 0
+
+    if (this._height !== height) {
+      this.zafResize(width, height)
+      this._height = height
+    }
+  }
```
Comment (adrianbonnici) [inline src/javascripts/app.js:147]: To ensure that we don't invoke a ZAF resize for the same body `height`.

File: .github/pull_request_template.md (hunk 1 above)
Comment (dadah89) [reply .github/pull_request_template.md:32]: sounds good, thanks!

**Comment by sostopher** [comment]
@adrianbonnici Can you please bump the version in `package.json` and `manifest.json`? :)

**Pull request merged**